src/oic/oic/message.py

```diff
@@ -258,6 +259,50 @@ def check_char_set(string, allowed):
             raise NotAllowedValue("'%c' not in the allowed character set" % c)
 
 
+def verify_id_token(instance, check_hash=False, **kwargs):
+    # Try to decode the JWT, checks the signature
+    args = {}
+    for arg in ["key", "keyjar", "algs", "sender"]:
+        try:
+            args[arg] = kwargs[arg]
+        except KeyError:
+            pass
+
+    _jws = str(instance["id_token"])
+    _packer = JWT()
+    _body = _packer.unpack(_jws).payload()
+    if 'keyjar' in kwargs:
+        try:
+            if _body['iss'] not in kwargs['keyjar']:
+                raise ValueError('Unknown issuer')
+        except KeyError:
+            pass
```

Comment:
What is the purpose of this pass? What are we supposed to do if the `iss` is missing in the message?

This piece of code is also untested.

Reply:
Dealt with this.